easybuild/easyconfigs/m/MPICH/MPICH-3.2-GCC-4.9.3.eb

```diff
@@ -0,0 +1,19 @@
+name = 'MPICH'
+version = '3.2'
+
+homepage = 'http://www.mpich.org/'
+description = """MPICH v3.x is an open source high-performance MPI 3.0 implementation.
+It does not support InfiniBand (use MVAPICH2 with InfiniBand devices)."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.3'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.mpich.org/static/tarballs/%(version)s']
+
+sanity_check_paths = {
+    'files': ['bin/mpiexec', 'bin/mpirun',
+              'include/mpi.h', 'include/mpif.h'],
+    'dirs': [],
+}
```

Comment:
Right, this is not a derivative, but the MPICH easyblock, that is the base class for `EB_MVAPICH2`, has by default the common ones. Whatever is not common (like `mpiexec`), goes in the `sanity_check_paths` of the easyconfig.

I don't think we need `custom_sanity_check_paths`, whatever is in `sanity_check_paths` is transparently added to the common checks.